Dockerfile

```diff
@@ -0,0 +1,31 @@
+FROM python:2.7
+
+# based on https://github.com/pfichtner/docker-mqttwarn
+
+# install python libraries (TODO: any others?)
```

Comment:
Are there others?

Reply:
lots, depending on the services people want to use. Ignore for now, as this will be changing later as @amotl already indicated.